Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -345,8 +345,10 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         version += 1
         attributes.merge(rumCommandAttributes: command.attributes)
 
-        let timeSpent = command.time.timeIntervalSince(viewStartTime)
+        // RUMM-1779 Keep view active as long as we have ongoing resources
+        let isActive = isActiveView || !resourceScopes.isEmpty
```

Comment:
why do we wait for resources only? 
in theory, there can be ongoing actions as well.?

Reply:
It was my question from our previous daily and only resources will keep the view active, it's the same logic to [complete a view scope](https://github.com/DataDog/dd-sdk-ios/blob/master/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift#L213L216)